### ISSUE TITLE
Show author's name

### ISSUE THREAD

**Issue by orbifx** [fabienwang/Ghost-Flat#8]
I'm using this theme in a multi-authored blog. Can we enable display of author's name?

**Comment by fabienwang** [comment]
Hi. Thanks for your concern about that.
The theme was originally made for 1 author but since you request it i'm working on an adaptation to display multiple authors.
This is a huge work since it requires me to also create author's page But i'll keep you informed.

**Comment by orbifx** [comment]
Anyway I can help?
-- 
    SP

**Comment by fabienwang** [comment]
Sure. You can tell me what you think about this design:
![capture](https://cloud.githubusercontent.com/assets/8741873/9713674/4a1545d0-5553-11e5-977e-eceb868d8e3d.JPG)

Author names would be displayed at last within the post tags

I still have to design the author page & add the author in the header of post page.

Thanks for your feedback

**Comment by orbifx** [comment]
Overall looks great. Why not have the name first? One can always then be certain the first entry is always the author.

**Comment by fabienwang** [comment]
Good idea I'll do that. Maybe i'll add some extra space to more distinguish tags from author names too.

**Comment by orbifx** [comment]
Makes sense.

**Comment by fabienwang** [comment]
Now it's a bit better.

![author-before](https://cloud.githubusercontent.com/assets/8741873/9776950/5ff45596-575e-11e5-9622-cce94b94f836.png)

**Comment by orbifx** [comment]
Nice. I will pull, check and get back to you.

**Comment by orbifx** [comment]
Looks great on the main page. Can it also be added on the post page?

**Comment by orbifx** [comment]
Also, minor issue if the window size is a limited, the tags overlap:
![ghost-flag-author](https://cloud.githubusercontent.com/assets/1543042/9784640/3feccf3a-57a4-11e5-8ff5-a101b67248a8.png)

**Comment by fabienwang** [comment]
@orbifx can you give me your screen size or what device it is because I did something (changing line-height) for that matter. It seems what I did was not sufficient.

**Comment by orbifx** [comment]
My browsers width is about 1100px wide when this occurs. Using Seamonkey but happens in Firefox too.

**Comment by fabienwang** [comment]
Hello, sorry for the long time since this issue, I just submitted a bug to the upstream regarding author page where cover image and author image are mixed up.
https://github.com/TryGhost/Ghost/issues/6221

Once this issue is resolve i'll make a release and close this issue.
You can already use it if you don't use both author cover

**Comment by orbifx** [comment]
I presume fc7b4b6e0fbfe726042c556ffc8952397a1e2f33 fixed this?
Downloaded it and avatars seem well aligned.

**Comment by fabienwang** [comment]
Hey @orbifx hmmm yes It was fixed upstream few months ago (actually, just after I report the issue to them ).

I'll make a new release and close this issue in a few days. Thanks for your support @orbifx

**Comment by orbifx** [comment]
Thank you for taking care of this.
-- 
	SP
